src/main/java/com/videoclub/suriken/service/impl/MovieServiceImpl.java

```diff
@@ -0,0 +1,78 @@
+package com.videoclub.suriken.service.impl;
+
+import com.videoclub.suriken.model.Movie;
+import com.videoclub.suriken.model.MovieRenter;
+import com.videoclub.suriken.repository.MovieRepository;
+import com.videoclub.suriken.repository.RenterRepository;
+import com.videoclub.suriken.service.MovieService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class MovieServiceImpl implements MovieService {
+
+    private final MovieRepository movieRepository;
+    private final RenterRepository renterRepository;
+
+    public MovieServiceImpl(MovieRepository movieRepository, RenterRepository renterRepository) {
+        this.movieRepository = movieRepository;
+        this.renterRepository = renterRepository;
+    }
+
+
```

Comment:
Obicno se pazi na to da je u kodu nema nepotrebnih praznih linija. Ovde recimo ima jedna viska. Neki ljudi jako paze na to (ja) hahaha

Reply:
Hahah e i ja, ekstra. Neces videti mnogo takvih gresaka!